sdks/python/apache_beam/utils/python_callable.py

```diff
@@ -106,6 +106,19 @@ def load_from_script(source):
     exec('\n'.join(lines), exec_globals)
     return exec_globals[name]
 
+  def default_label(self):
+    src = self._source.strip()
+    last_line = src.split('\n')[-1]
+    if last_line[0] != ' ' and len(last_line) < 72:
```

Comment:
Where does 72 come from ?

Reply:
An arbitrary limit for "this is getting to big to show on one line." We wouldn't want unboundedly-large stage names with huge chunks of source code in them, but if it's small like "lambda x: 2 * x" that's nice.